AdminCore.Services/EventService.cs

```diff
@@ -614,10 +665,10 @@ private static bool IsNotPublicHoliday(Event eventToUpdate)
     private static int AutoApproveEventsNotNeedingAdminApproval(EventTypes eventTypes)
     {
       var eventStatusId = (int)EventStatuses.AwaitingApproval;
-      if (eventTypes == EventTypes.WorkingFromHome || eventTypes == EventTypes.Sickness)
-      {
-        eventStatusId = (int)EventStatuses.Approved;
-      }
+//      if (eventTypes == EventTypes.WorkingFromHome || eventTypes == EventTypes.Sickness)
```

Comment:
what's this?

Reply:
The EventService doesn't handle autoapproving events anymore. The Workflow Service should handle this (But it doesn't at the minute because we're waiting on more detail on the specs)

Reply:
I've removed the code in the EventService